test_autoscaler::AutoscalingTest.testDockerFileMountsRemoved Windows CI Failure

<!--Please include [tune], [rllib], [autoscaler] etc. in the issue title if relevant-->

### What is the problem?

*Ray version and other system information (Python version, TensorFlow version, OS):*
```
AILED: //python/ray/tests:test_autoscaler (Summary)
      C:/users/runneradmin/_bazel_runneradmin/vlncal46/execroot/com_github_ray_project_ray/bazel-out/x64_windows-opt/testlogs/python/ray/tests/test_autoscaler/test.log
      C:/users/runneradmin/_bazel_runneradmin/vlncal46/execroot/com_github_ray_project_ray/bazel-out/x64_windows-opt/testlogs/python/ray/tests/test_autoscaler/test_attempts/attempt_1.log
      C:/users/runneradmin/_bazel_runneradmin/vlncal46/execroot/com_github_ray_project_ray/bazel-out/x64_windows-opt/testlogs/python/ray/tests/test_autoscaler/test_attempts/attempt_2.log
INFO: From Testing //python/ray/tests:test_autoscaler:
==================== Test output for //python/ray/tests:test_autoscaler:
============================= test session starts =============================
platform win32 -- Python 3.7.9, pytest-5.4.3, py-1.10.0, pluggy-0.13.1 -- C:\hostedtoolcache\windows\Python\3.7.9\x64\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\runneradmin\AppData\Local\Temp\Bazel.runfiles_rjyqimfr\runfiles\com_github_ray_project_ray
plugins: asyncio-0.14.0, rerunfailures-9.1.1, sugar-0.9.4, timeout-1.4.2
collecting ... collected 42 items

::LoadMetricsTest::testDebugString PASSED                                [  2%]
::LoadMetricsTest::testHeartbeat PASSED                                  [  4%]
::AutoscalingTest::testAggressiveAutoscaling 2021-01-05 06:36:32,054	SUCC updater.py:241 -- Success.
2021-01-05 06:36:32,054	INFO log_timer.py:27 -- NodeUpdater: 1: Got remote shell  [LogTimer=13ms]
2021-01-05 06:36:32,054	INFO updater.py:318 -- Updating cluster configuration. [hash=a4a92c863e67620ebfe00d7b8e6420316984cf91]
2021-01-05 06:36:32,054	INFO updater.py:322 -- New status: syncing-files
2021-01-05 06:36:32,054	INFO updater.py:208 -- [2/7] Processing file mounts
2021-01-05 06:36:32,054	INFO updater.py:225 -- [3/7] No worker file mounts to sync
2021-01-05 06:36:32,054	INFO updater.py:333 -- New status: setting-up
2021-01-05 06:36:32,054	INFO updater.py:338 -- [4/7] Running initialization commands
2021-01-05 06:36:32,054	VINFO command_runner.py:508 -- Running `export RAY_HEAD_IP=172.0.0.0; init_cmd`
2021-01-05 06:36:32,054	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=172.0.0.0; init_cmd)'`
2021-01-05 06:36:32,054	INFO log_timer.py:27 -- NodeUpdater: 1: Initialization commands succeeded [LogTimer=0ms]
2021-01-05 06:36:32,055	INFO updater.py:375 -- [5/7] Initalizing command runner
2021-01-05 06:36:32,055	VINFO command_runner.py:508 -- Running `command -v docker || echo 'NoExist'`
2021-01-05 06:36:32,055	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (command -v docker || echo '"'"'NoExist'"'"')'`
2021-01-05 06:36:32,055	VINFO command_runner.py:508 -- Running `docker pull example`
2021-01-05 06:36:32,055	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker pull example)'`
2021-01-05 06:36:32,055	VINFO command_runner.py:508 -- Running `docker inspect -f '{{.State.Running}}' mock || true`
2021-01-05 06:36:32,055	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker inspect -f '"'"'{{.State.Running}}'"'"' mock || true)'`
2021-01-05 06:36:32,055	VINFO command_runner.py:508 -- Running `docker inspect -f '{{json .Config.Env}}' example`
2021-01-05 06:36:32,055	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker inspect -f '"'"'{{json .Config.Env}}'"'"' example)'`
2021-01-05 06:36:32,056	VINFO command_runner.py:508 -- Running `docker info -f '{{.Runtimes}}' `
2021-01-05 06:36:32,056	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker info -f '"'"'{{.Runtimes}}'"'"' )'`
2021-01-05 06:36:32,056	VINFO command_runner.py:508 -- Running `cat /proc/meminfo || true`
2021-01-05 06:36:32,056	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (cat /proc/meminfo || true)'`
2021-01-05 06:36:32,056	VINFO command_runner.py:508 -- Running `docker run --rm --name mock -d -it  -e LC_ALL=C.UTF-8 -e LANG=C.UTF-8  --net=host example bash`
2021-01-05 06:36:32,056	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker run --rm --name mock -d -it  -e LC_ALL=C.UTF-8 -e LANG=C.UTF-8  --net=host example bash)'`
2021-01-05 06:36:32,056	INFO updater.py:384 -- [6/7] Running setup commands
2021-01-05 06:36:32,056	INFO updater.py:404 -- (0/2) export RAY_HEAD_IP=172.0.0.0; setup_cmd
2021-01-05 06:36:32,056	VINFO command_runner.py:508 -- Running `docker exec -it  mock /bin/bash -c 'bash --login -c -i '"'"'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=172.0.0.0; setup_cmd)'"'"'' `
2021-01-05 06:36:32,057	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker exec -it  mock /bin/bash -c '"'"'bash --login -c -i '"'"'"'"'"'"'"'"'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=172.0.0.0; setup_cmd)'"'"'"'"'"'"'"'"''"'"' )'`
2021-01-05 06:36:32,057	INFO updater.py:404 -- (1/2) export RAY_HEAD_IP=172.0.0.0; worker_setup_cmd
2021-01-05 06:36:32,057	VINFO command_runner.py:508 -- Running `docker exec -it  mock /bin/bash -c 'bash --login -c -i '"'"'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=172.0.0.0; worker_setup_cmd)'"'"'' `
2021-01-05 06:36:32,057	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker exec -it  mock /bin/bash -c '"'"'bash --login -c -i '"'"'"'"'"'"'"'"'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=172.0.0.0; worker_setup_cmd)'"'"'"'"'"'"'"'"''"'"' )'`
2021-01-05 06:36:32,057	INFO log_timer.py:27 -- NodeUpdater: 1: Setup commands succeeded [LogTimer=1ms]
2021-01-05 06:36:32,057	INFO updater.py:424 -- [7/7] Starting the Ray runtime
2021-01-05 06:36:32,057	VINFO command_runner.py:508 -- Running `docker exec -it  mock /bin/bash -c 'bash --login -c -i '"'"'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=172.0.0.0; start_ray_worker)'"'"'' `
2021-01-05 06:36:32,057	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker exec -it  mock /bin/bash -c '"'"'bash --login -c -i '"'"'"'"'"'"'"'"'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=172.0.0.0; start_ray_worker)'"'"'"'"'"'"'"'"''"'"' )'`
2021-01-05 06:36:32,057	INFO log_timer.py:27 -- NodeUpdater: 1: Ray start commands succeeded [LogTimer=0ms]
2021-01-05 06:36:32,058	INFO log_timer.py:27 -- NodeUpdater: 1: Applied config a4a92c863e67620ebfe00d7b8e6420316984cf91  [LogTimer=18ms]
2021-01-05 06:36:32,058	INFO updater.py:157 -- New status: up-to-date
PASSED                      [  7%]
::AutoscalingTest::testAutodetectResources PASSED                        [  9%]
::AutoscalingTest::testAutoscalerConfigValidationFailNotFatal 2021-01-05 06:36:32,483	INFO updater.py:333 -- New status: setting-up
2021-01-05 06:36:32,483	INFO updater.py:338 -- [4/7] Running initialization commands
2021-01-05 06:36:32,483	VINFO command_runner.py:508 -- Running `export RAY_HEAD_IP=10.1.0.4; init_cmd`
2021-01-05 06:36:32,483	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=10.1.0.4; init_cmd)'`
2021-01-05 06:36:32,483	INFO log_timer.py:27 -- NodeUpdater: 1: Initialization commands succeeded [LogTimer=0ms]
2021-01-05 06:36:32,484	INFO updater.py:375 -- [5/7] Initalizing command runner
2021-01-05 06:36:32,484	VINFO command_runner.py:508 -- Running `command -v docker || echo 'NoExist'`
2021-01-05 06:36:32,484	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (command -v docker || echo '"'"'NoExist'"'"')'`
2021-01-05 06:36:32,484	VINFO command_runner.py:508 -- Running `docker pull example`
2021-01-05 06:36:32,484	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker pull example)'`
2021-01-05 06:36:32,484	VINFO command_runner.py:508 -- Running `docker inspect -f '{{.State.Running}}' mock || true`
2021-01-05 06:36:32,484	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker inspect -f '"'"'{{.State.Running}}'"'"' mock || true)'`
2021-01-05 06:36:32,484	VINFO command_runner.py:508 -- Running `docker inspect -f '{{json .Config.Env}}' example`
2021-01-05 06:36:32,484	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.1 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker inspect -f '"'"'{{json .Config.Env}}'"'"' example)'`
2021-01-05 06:36:32,485	INFO log_timer.py:27 -- NodeUpdater: 1: Applied config a4a92c863e67620ebfe00d7b8e6420316984cf91  [LogTimer=4ms]
2021-01-05 06:36:32,485	ERR updater.py:128 -- New status: update-failed
2021-01-05 06:36:32,485	ERR updater.py:130 -- !!!
2021-01-05 06:36:32,485	VERR updater.py:136 -- {'msg': 'Expecting value', 'doc': 'command-output', 'pos': 0, 'lineno': 1, 'colno': 1}
2021-01-05 06:36:32,485	ERR updater.py:138 -- Expecting value: line 1 column 1 (char 0)
2021-01-05 06:36:32,485	ERR updater.py:140 -- !!!
PASSED     [ 11%]Docker not installed. You can install Docker by adding the following commands to 'initialization_commands':
curl -fsSL https://get.docker.com -o get-docker.sh
sudo sh get-docker.sh
sudo usermod -aG docker $USER
sudo systemctl restart docker -f
2021-01-05 06:36:32,488	VINFO command_runner.py:508 -- Running `uptime`
2021-01-05 06:36:32,488	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=5s ubuntu@172.0.0.0 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (uptime)'`
2021-01-05 06:36:32,488	SUCC updater.py:241 -- Success.
2021-01-05 06:36:32,488	INFO log_timer.py:27 -- NodeUpdater: 0: Got remote shell  [LogTimer=6ms]
2021-01-05 06:36:32,488	INFO updater.py:318 -- Updating cluster configuration. [hash=a4a92c863e67620ebfe00d7b8e6420316984cf91]
2021-01-05 06:36:32,488	INFO updater.py:322 -- New status: syncing-files
2021-01-05 06:36:32,488	INFO updater.py:208 -- [2/7] Processing file mounts
2021-01-05 06:36:32,488	INFO updater.py:225 -- [3/7] No worker file mounts to sync
2021-01-05 06:36:32,488	INFO updater.py:333 -- New status: setting-up
2021-01-05 06:36:32,489	INFO updater.py:338 -- [4/7] Running initialization commands
2021-01-05 06:36:32,489	VINFO command_runner.py:508 -- Running `export RAY_HEAD_IP=10.1.0.4; init_cmd`
2021-01-05 06:36:32,489	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ConnectTimeout=120s ubuntu@172.0.0.0 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (export RAY_HEAD_IP=10.1.0.4; init_cmd)'`
2021-01-05 06:36:32,489	INFO log_timer.py:27 -- NodeUpdater: 0: Initialization commands succeeded [LogTimer=0ms]
2021-01-05 06:36:32,489	INFO updater.py:375 -- [5/7] Initalizing command runner
2021-01-05 06:36:32,489	VINFO command_runner.py:508 -- Running `command -v docker || echo 'NoExist'`
2021-01-05 06:36:32,489	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.0 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (command -v docker || echo '"'"'NoExist'"'"')'`

2021-01-05 06:36:32,490	VINFO command_runner.py:508 -- Running `docker pull example`
2021-01-05 06:36:32,490	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.0 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker pull example)'`
2021-01-05 06:36:32,490	VINFO command_runner.py:508 -- Running `docker inspect -f '{{.State.Running}}' mock || true`
2021-01-05 06:36:32,490	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.0 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker inspect -f '"'"'{{.State.Running}}'"'"' mock || true)'`
2021-01-05 06:36:32,490	VINFO command_runner.py:508 -- Running `docker inspect -f '{{json .Config.Env}}' example`
2021-01-05 06:36:32,490	VVINFO command_runner.py:511 -- Full command is `ssh -tt -i nul -o StrictHostKeyChecking=no -o UserKnownHostsFile=nul -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ControlMaster=auto -o ControlPath=/tmp/ray_ssh_325c223333/c21f969b5f/%C -o ControlPersist=10s -o ConnectTimeout=120s ubuntu@172.0.0.0 bash --login -c -i 'true && source ~/.bashrc && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore && (docker inspect -f '"'"'{{json .Config.Env}}'"'"' example)'`
2021-01-05 06:36:32,491	INFO log_timer.py:27 -- NodeUpdater: 0: Applied config a4a92c863e67620ebfe00d7b8e6420316984cf91  [LogTimer=11ms]
2021-01-05 06:36:32,491	ERR updater.py:128 -- New status: update-failed
2021-01-05 06:36:32,491	ERR updater.py:130 -- !!!
2021-01-05 06:36:32,491	VERR updater.py:136 -- {'msg': 'Expecting value', 'doc': 'command-output', 'pos': 0, 'lineno': 1, 'colno': 1}
2021-01-05 06:36:32,491	ERR updater.py:138 -- Expecting value: line 1 column 1 (char 0)
2021-01-05 06:36:32,491	ERR updater.py:140 -- !!!
::AutoscalingTest::testConfiguresNewNodes PASSED                         [ 14%]
::AutoscalingTest::testConfiguresOutdatedNodes PASSED                    [ 16%]
::AutoscalingTest::testContinuousFileMounts SKIPPED                      [ 19%]
::AutoscalingTest::testDelayedLaunch PASSED                              [ 21%]
::AutoscalingTest::testDelayedLaunchWithMinWorkers PASSED                [ 23%]
::AutoscalingTest::testDockerFileMountsAdded SKIPPED                     [ 26%]
::AutoscalingTest::testDockerFileMountsRemoved FAILED                    [ 28%]
::AutoscalingTest::testDynamicScaling PASSED                             [ 30%]
::AutoscalingTest::testExternalNodeScaler PASSED                         [ 33%]
::AutoscalingTest::testExternalNodeScalerWrongImport PASSED              [ 35%]
::AutoscalingTest::testExternalNodeScalerWrongModuleFormat PASSED        [ 38%]
::AutoscalingTest::testFileMountsNonContinuous PASSED                    [ 40%]
::AutoscalingTest::testGetOrCreateHeadNode SKIPPED                       [ 42%]
::AutoscalingTest::testGetOrCreateHeadNodeFromStopped SKIPPED            [ 45%]
::AutoscalingTest::testIgnoresCorruptedConfig PASSED                     [ 47%]
::AutoscalingTest::testInitialWorkers PASSED                             [ 50%]
::AutoscalingTest::testLaunchConfigChange PASSED                         [ 52%]
::AutoscalingTest::testLaunchNewNodeOnOutOfBandTerminate PASSED          [ 54%]
::AutoscalingTest::testLegacyYamlWithRequestResources PASSED             [ 57%]
::AutoscalingTest::testMaxFailures PASSED                                [ 59%]
::AutoscalingTest::testMultiNodeReuse PASSED                             [ 61%]
::AutoscalingTest::testRecoverUnhealthyWorkers PASSED                    [ 64%]
::AutoscalingTest::testReportsConfigFailures PASSED                      [ 66%]
::AutoscalingTest::testRsyncCommandWithDocker SKIPPED                    [ 69%]
::AutoscalingTest::testRsyncCommandWithoutDocker SKIPPED                 [ 71%]
::AutoscalingTest::testScaleUp PASSED                                    [ 73%]
::AutoscalingTest::testScaleUpBasedOnLoad PASSED                         [ 76%]
::AutoscalingTest::testSetupCommandsWithNoNodeCaching PASSED             [ 78%]
::AutoscalingTest::testSetupCommandsWithStoppedNodeCachingDocker PASSED  [ 80%]
::AutoscalingTest::testSetupCommandsWithStoppedNodeCachingNoDocker PASSED [ 83%]
::AutoscalingTest::testTargetUtilizationFraction PASSED                  [ 85%]
::AutoscalingTest::testTerminateOutdatedNodesGracefully PASSED           [ 88%]
::AutoscalingTest::testUnmanagedNodes PASSED                             [ 90%]
::AutoscalingTest::testUnmanagedNodes2 PASSED                            [ 92%]
::AutoscalingTest::testUpdateThrottling PASSED                           [ 95%]
::AutoscalingTest::testValidateDefaultConfig PASSED                      [ 97%]
::AutoscalingTest::testValidation PASSED                                 [100%]

================================== FAILURES ===================================
_________________ AutoscalingTest.testDockerFileMountsRemoved _________________

self = <com_github_ray_project_ray.python.ray.tests.test_autoscaler.AutoscalingTest testMethod=testDockerFileMountsRemoved>

    def testDockerFileMountsRemoved(self):
        config = copy.deepcopy(SMALL_CLUSTER)
        config["file_mounts"] = {}
        config_path = self.write_config(config)
        self.provider = MockProvider()
        runner = MockProcessRunner()
        mounts = [{
            "Type": "bind",
            "Source": "/sys",
            "Destination": "/sys",
            "Mode": "ro",
            "RW": False,
            "Propagation": "rprivate"
        }]
        runner.respond_to_call("json .Mounts", [json.dumps(mounts)])
        # Two initial calls to docker cp, +1 more call during run_init
        runner.respond_to_call(".State.Running",
                               ["false", "false", "true", "true"])
        runner.respond_to_call("json .Config.Env", ["[]"])
        commands.get_or_create_head_node(
            config,
            printable_config_file=config_path,
            no_restart=False,
            restart_only=False,
            yes=True,
            override_cluster_name=None,
            _provider=self.provider,
>           _runner=runner)

\\?\C:\Users\RUNNER~1\AppData\Local\Temp\Bazel.runfiles_rjyqimfr\runfiles\com_github_ray_project_ray\python\ray\tests\test_autoscaler.py:575: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
d:\a\ray\ray\python\ray\autoscaler\_private\commands.py:710: in get_or_create_head_node
    cli_logger.abort("Failed to setup head node.")
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <ray.autoscaler._private.cli_logger._CliLogger object at 0x0000023711F1A708>
msg = 'Failed to setup head node.', exc = None, args = (), kwargs = {}
exc_cls = <class 'click.exceptions.ClickException'>

    def abort(self,
              msg: Optional[str] = None,
              *args: Any,
              exc: Any = None,
              **kwargs: Any):
        """Prints an error and aborts execution.
    
        Print an error and throw an exception to terminate the program
        (the exception will not print a message).
        """
        if msg is not None:
            self._error(msg, *args, _level_str="PANIC", **kwargs)
    
        if exc is not None:
            raise exc
    
        exc_cls = click.ClickException
        if self.pretty:
            exc_cls = SilentClickException
>       raise exc_cls("Exiting due to cli_logger.abort()")
E       click.exceptions.ClickException: Exiting due to cli_logger.abort()

d:\a\ray\ray\python\ray\autoscaler\_private\cli_logger.py:585: ClickException
---------------------------- Captured stderr call -----------------------------
Exception in thread Thread-57:
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "d:\a\ray\ray\python\ray\autoscaler\_private\updater.py", line 124, in run
    self.do_update()
  File "d:\a\ray\ray\python\ray\autoscaler\_private\updater.py", line 324, in do_update
    self.rsync_up, step_numbers=(1, NUM_SETUP_STEPS))
  File "d:\a\ray\ray\python\ray\autoscaler\_private\updater.py", line 210, in sync_file_mounts
    do_sync(remote_path, local_path)
  File "d:\a\ray\ray\python\ray\autoscaler\_private\updater.py", line 180, in do_sync
    assert os.path.exists(local_path), local_path
AssertionError: nul

=========================== short test summary info ===========================
FAILED ::AutoscalingTest::testDockerFileMountsRemoved - click.exceptions.Clic...
================== 1 failed, 35 passed, 6 skipped in 12.58s ===================
Exception in thread Thread-156:
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "d:\a\ray\ray\python\ray\autoscaler\_private\updater.py", line 124, in run
    self.do_update()
  File "d:\a\ray\ray\python\ray\autoscaler\_private\updater.py", line 379, in do_update
    sync_run_yet=True)
  File "d:\a\ray\ray\python\ray\autoscaler\_private\command_runner.py", line 801, in run_init
    for env_var in json.loads(image_env):
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
https://github.com/ray-project/ray/runs/1647962851
### Reproduction (REQUIRED)
Please provide a short code snippet (less than 50 lines if possible) that can be copy-pasted to reproduce the issue. The snippet should have **no external library dependencies** (i.e., use fake or mock data / environments):

If the code snippet cannot be run by itself, the issue will be closed with "needs-repro-script".

- [ ] I have verified my script runs in a clean environment and reproduces the issue.
- [ ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/master/installation.html).

--- a/python/ray/tests/test_autoscaler.py
+++ b/python/ray/tests/test_autoscaler.py
@@ -545,7 +545,7 @@ class AutoscalingTest(unittest.TestCase):
             f"docker cp {docker_mount_prefix}/~/ray_bootstrap_config.yaml"
         runner.assert_has_call("1.2.3.4", pattern=pattern_to_assert)
 
-    @unittest.skipIf(sys.platform == "win32", "Failing on Windows.")s
+    @unittest.skipIf(sys.platform == "win32", "Failing on Windows.")
     def testDockerFileMountsRemoved(self):
         config = copy.deepcopy(SMALL_CLUSTER)
         config["file_mounts"] = {}

--- a/python/ray/tests/test_autoscaler.py
+++ b/python/ray/tests/test_autoscaler.py
@@ -545,6 +545,7 @@ class AutoscalingTest(unittest.TestCase):
             f"docker cp {docker_mount_prefix}/~/ray_bootstrap_config.yaml"
         runner.assert_has_call("1.2.3.4", pattern=pattern_to_assert)
 
+    @unittest.skipIf(sys.platform == "win32", "Failing on Windows.")s
     def testDockerFileMountsRemoved(self):
         config = copy.deepcopy(SMALL_CLUSTER)
         config["file_mounts"] = {}

